release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+k6 `v0.49.0` is here 🎉! This release includes:
+
+- Add a built-in [web dashboard](https://grafana.com/docs/k6/latest/results-output/web-dashboard/) that displays test results in real time.
+- Add `clear` functionality to the browser module's `locator` classes.
+- Merge the gRPC experimental module back into the gRPC core module.
+- Add the ability to get the selection from an element in `k6/html`. 
+- Collect internal modules and outputs used by a script.
+- Get `k6/experimental/timers` ready to be stabilized.
```

Comment:
I'm not sure if I'm right here; it's probably better to check with @heitortsergent, but is `This release includes:` aligned with the following list? 

I mean, should it be something like:

```suggestion
k6 `v0.49.0` is here 🎉! This release:

- Adds a built-in [web dashboard](https://grafana.com/docs/k6/latest/results-output/web-dashboard/) that displays test results in real time.
- Introduces `clear` functionality to the browser module's `locator` classes.
- Merges the gRPC experimental module back into the gRPC core module.
- Enables the ability to get the selection from an element in `k6/html`.
- Collects internal modules and outputs used by a script.
- Prepares `k6/experimental/timers` for stabilization.
```